src/sentry/templates/sentry/js-sdk-loader.js.tmpl

```diff
@@ -1,5 +1,24 @@
 {% load sentry_helpers %}// Sentry Loader
 (function(_window, _document, _script, _onerror, _onunhandledrejection) {
+  var sentryCallQueue = [];
+  _window.Sentry = {
```

Comment:
I'd do `_window.Sentry = _window.Sentry || { ... }` juuust in case someone includes Sentry somewhere in their webpack config or some other magic thing and they shoot themselves in the foot.